ui/src/features/get-tests.js

```diff
@@ -118,13 +128,36 @@ class getTests extends React.Component {
       this.setState({ openViewTest: data });
     };
 
+    updateTestActionError = ({ isError, errorMessage }) => {
+      this.setState({
+        ...this.state,
```

Comment:
this.setState update only the keys that you send, so  you don't need   "...this.state "

Reply:
done again!